src/utils/format.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Uses Intl.NumberFormat style='decimal' to add thousands separator to a
+ * displayed number
+ *
+ * @param value
+ */
+export const formatNumber = (value: string | number | bigint | null) => {
+  const numValue = !value
+    ? 0
+    : typeof value === 'string'
+      ? parseFloat(value)
+      : typeof value === 'bigint'
+        ? Number(value)
+        : value;
+
+  return new Intl.NumberFormat(undefined, {
```

Comment:
Allegedly, `undefined` should pick the browsers locale. I tried using a VPN in European locales to see `$ 100.00,00` but was unsuccessful.

Reply:
Right, browser locale is dependent upon OS configuration but independent of network. But your understanding about `undefined` is correct.